justfile

```diff
@@ -449,6 +455,13 @@ _devcontainer-build tag target='':
         --target='{{ target }}' \
         --{{ if devcontainer-build-mode == "push" { "push" } else { "load" } }}
 
+##
+## tools
+##
+
+markdownlint:
```

Comment:
wondering if this should be `markdown-lint` for consistency with the `go-lint` recipe? OTTOH, `markdownlint` is the name of the tool, so idk...

Reply:
I like this. Simplified to `md-lint`. Added sh-lint, etc.